src/decode.rs

```diff
@@ -4185,104 +4185,40 @@ unsafe fn decode_b(
             assert!(b.drl_idx() >= NEAREST_DRL as u8 && b.drl_idx() <= NEARISH_DRL as u8);
 
             has_subpel_filter = imin(bw4, bh4) == 1 || b.inter_mode() != GLOBALMV_GLOBALMV as u8;
```

Comment:
In C, this line is after the macro and right before `assign_comp_mv(0);`.  Could you move it there?

Reply:
Oh, I missed that it uses it in the closure.  In that case, this is the right way.